docs/matchers/dialogflow.md

```diff
@@ -1,13 +1,13 @@
-# api.ai Matcher
+# Dialogflow Matcher (previously api.ai)
 
-[api.ai](https://api.ai) is an NLP API for matching strings to [intents](https://docs.api.ai/docs/concept-intents) or [actions](https://docs.api.ai/docs/concept-actions). Intents are created on the api.ai website.
+[Dialogflow](https://dialogflow.com/) is an NLP API for matching strings to [intents](https://dialogflow.com/docs/intents) or [actions](https://dialogflow.com/docs/concept-actions). Intents are created on the dialogflow website.
 
 ## Example 1
 
 ```python
-from opsdroid.matchers import match_apiai_action
+from opsdroid.matchers import match_dialogflow_action
 
-@match_apiai_action('mydomain.myaction')
+@match_dialogflow_action('mydomain.myaction')
 async def mySkill(opsdroid, config, message):
```

Comment:
I know I wrote this example originally but it should be `my_skill` not `mySkill`, could you change it while you're at it?